### PR TITLE
console.dump implemented for ios

### PR DESCRIPTION
**Pull request by enchev**
Implemenetd #805

### CODE REVIEW

**Comment by enchev** [comment]
run ci

File: globals/globals.ts [hunk 1]
```diff
@@ -30,9 +30,12 @@ if (typeof global.__decorate !== "function") {
...
 
-// Temporary workaround for console in iOS. We will use runtime console instead our implementation.
-if (types.isUndefined(global.NSObject)) {
-    global.console = new consoleModule.Console();
+var console = new consoleModule.Console();
+
+if (global.android) {
+    global.console.dump = function (args) { console.dump(args); };
+} else {
+    global.console = console;
```
Comment (fealebenpae) [inline globals/globals.ts]: Wouldn't replacing the entire console object hobble the iOS Inspector?

**Comment by fealebenpae** [comment]
:+1:

**Comment by enchev** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Pull request merged**